packages/markdown-slate/src/__snapshots__/SlateTransformer.test.js.snap

```diff
@@ -236,104 +236,26 @@ Object {
                 "object": "text",
                 "text": "Acceptance of Delivery. ",
               },
-              Object {
-                "children": Array [
-                  Object {
-                    "object": "text",
-                    "text": "\\"Party A\\"",
-                  },
-                ],
-                "data": Object {
-                  "id": "shipper",
-                },
-                "object": "inline",
-                "type": "variable",
-              },
               Object {
                 "object": "text",
                 "text": " will be deemed to have completed its delivery obligations if in ",
               },
-              Object {
-                "children": Array [
-                  Object {
-                    "object": "text",
-                    "text": "\\"Party B\\"",
-                  },
-                ],
-                "data": Object {
-                  "id": "receiver",
-                },
-                "object": "inline",
-                "type": "variable",
-              },
               Object {
                 "object": "text",
                 "text": "'s opinion, the ",
               },
-              Object {
-                "children": Array [
-                  Object {
-                    "object": "text",
-                    "text": "\\"Widgets\\"",
-                  },
-                ],
-                "data": Object {
-                  "id": "deliverable",
-                },
-                "object": "inline",
-                "type": "variable",
-              },
               Object {
                 "object": "text",
                 "text": " satisfies the Acceptance Criteria, and ",
               },
-              Object {
```

Comment:
Something is going seriously wrong in those tests. All the inlines are disappearing. I am suspecting because the logic doesn't push the inlines in the new children list. Input children should always be pushed to the result.